packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -299,6 +303,66 @@ const assertNoImportedCssVariables = (
   }
 };
 
+/**
+ * Manipulates template literal so that it resembles
+ * CSS `property: value` declaration where
+ * - before: all the CSS _before_ the interpolation
+ * - after: all the CSS _after_ the interpolation
+ * @param expression
+ * @param quasi
+ */
+const moveCssPropertyInExpression = (
```

Comment:
Since this is fairly standalone, could this util be moved into its own file?

Reply:
- [ ] moved into its own file